libseshat/tests/simple.c

```diff
@@ -131,10 +131,50 @@ void test_all( void )
    
 }
 
+void test_none( void )
+{
+    char *url;
+    int seshat_pid;
+    int seshat_service;
+    int cnt = 0;
+    char test_buffer[64];
+    char test_url[256];
+    char test_service[32];
+
+    sprintf(test_buffer, "WebPa01");
+    url = seshat_discover(test_buffer);
+    CU_ASSERT(NULL == url);
+    free(url);
+
```

Comment:
This is a different test, _test_none_. So, this is expected behavior. _test_all_ is still present.